phone-book/src/contact.js

```diff
@@ -0,0 +1,227 @@
+
+                    </table>
+                </main>
+
+                ${this.contactFooterComponent()}
+            </div>
+        `;
+    }
+
+    contactHeaderComponent(){
+        return `
+            <header class="container">
+                <form class="form-inline search-form row">
+                    <div class="form-group">
+                        <label class="sr-only" for="search">Search</label>
+                        <input type="text" class="form-control" id= "search" placeholder="Search">
+                    </div>
+                </form>
+            </header>
+        `
+    }
+
+    contactHeaderListComponent() {
+        return `
+            <thead>
+                <tr>
+                    <th>Name</th>
+                    <th>Last name</th>
+                    <th>Email</th>
+                </tr>
+            </thead>
+        `
+    }
+
+    contactListComponent() {
+        const contactList = document.createElement('tbody');
+        contactList.classList.add('contact-list');
+
+        users.forEach(user => {
+            const userFirstName = user[0].name;
+            const userLastName = user[0].lastName;
+            const userEmail = user[0].email;
+
+            contactList.innerHTML +=  `
+                <tr class="user">
+                    <td> ${userFirstName} </td>
+                    <td> ${userLastName} </td>
+                    <td> ${userEmail} </td>
+                </tr>
+            `
+        })
+
+        return contactList.outerHTML;
+    }
+
+    contactFooterComponent() {
+        return `
+            <footer class="container">
+            
+                <div class="row navigation-panel">
+                
+                    <div class="col-4 d-flex justify-content-center">
+                        <a href="#">
+                        <button class="navigation-button active">
+                            <span class="far fa-address-book icon active"></span>
+                        </button>
+                        </a>
+                    </div>
+                    
+                    <div class="col-4 d-flex justify-content-center">
+                        <a href="keypad.html">
+                        <button class="navigation-button">
+                            <span class="fas fa-tty icon"></span>
+                        </button>
+                        </a>
+                    </div>
+                    
+                    <div class="col-4 d-flex justify-content-center">
+                        <a href="#">
+                        <button class="navigation-button">
+                            <span class="fas fa-user-plus icon"></span>
+                        </button>
+                        </a>
+                    </div>
+                    
+                </div>
+                
+            </footer>
+        `
+    }
+
+    render() {
+        document.body.innerHTML += this.contactTemplate;
```

Comment:
Probably I don't see any reasons for class instance variable called contactTemplate.

You can just place HTML inside render a page and as I see `this.contactTemplate` used only here